utils/validators.js

```diff
@@ -111,4 +112,16 @@ module.exports = {
         }
       });
   },
+  validateRole: (role, errorMessage = 'Insufficient permission') => {
```

Comment:
this copy is a little too vague, see suggestion or anything along those lines
```suggestion
  validateRole: (role, errorMessage = 'You do not have sufficient permission to perform this action') => {
```

Reply:
Ok will do